tests/integration/hosts/maya/test_publish_in_maya/input/workfile/test_project_test_asset_test_task_v001.ma

```diff
@@ -0,0 +1,476 @@
+//Maya ASCII 2022 scene
+//Name: test_project_test_asset_test_task_v001.ma
+//Last modified: Thu, Sep 14, 2023 06:31:00 PM
+//Codeset: 1252
+requires maya "2022";
+requires -nodeType "polyDisc" "modelingToolkit" "0.0.0.0";
+requires "stereoCamera" "10.0";
+requires -nodeType "aiOptions" -nodeType "aiAOVDriver" -nodeType "aiAOVFilter" "mtoa" "5.2.2.1";
+requires -nodeType "simpleSelector" -nodeType "renderSetupLayer" -nodeType "renderSetup"
+		 -nodeType "collection" "renderSetup.py" "1.0";
+requires "stereoCamera" "10.0";
+currentUnit -l centimeter -a degree -t pal;
+fileInfo "application" "maya";
+fileInfo "product" "Maya 2022";
+fileInfo "version" "2022";
+fileInfo "cutIdentifier" "202205171752-c25c06f306";
+fileInfo "osv" "Windows 10 Pro v2009 (Build: 19044)";
+fileInfo "license" "education";
```

Comment:
Won't educational licensed files start adding all kinds of license warnings when running tests?

Reply:
Didnt get any issues running the tests here.